src/components/dialog/options.js

```diff
@@ -0,0 +1,20 @@
+import { translate } from 'react-i18next';
+import React from 'react';
+import grid from 'flexboxgrid/dist/flexboxgrid.css';
+import { Button, RedButton } from '../toolbox/buttons/button';
+
+import styles from './options.css';
+
+const Options = ({ firstButton, secondButton, text, title }) =>
+  (<div className={styles.optionsBody}>
+    <div className={styles.title}>{title}</div>
+    <br />
```

Comment:
I think is more maintainable if we add the spacings with css styles instead of <br /> markup elements